app/src/main/java/com/example/sharingang/items/ItemsViewModel.kt

```diff
@@ -0,0 +1,28 @@
+package com.example.sharingang.items
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+
+/**
+ * ItemsViewModel models the state of the fragment for viewing items
+ */
```

Comment:
In the comment, what does "fragment" mean?

Reply:
A fragment is the section of an activity used for displaying things. Concretely, this comment might be slightly misplaced, because there's no separate fragment from MainActivity right now, but the idea is that this state models that of the section of the activity responsible for displaying items.